pages/preferences.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: layout.njk
+title: Draft
```

Comment:
I didn't notice that this still says "Draft" in the title. Currently too lazy to fix this myself. Whoever commits first wins 😆